modules/activitypub/keypair_test.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2021 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package activitypub
+
+import (
+	"regexp"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestKeygen(t *testing.T) {
+	priv, pub, err := GenerateKeyPair()
+
+	assert.NoError(t, err)
+
+	assert.NotEmpty(t, priv)
+	assert.NotEmpty(t, pub)
+
+	assert.Regexp(t, regexp.MustCompile("^-----BEGIN RSA PRIVATE KEY-----.*"), priv)
+	assert.Regexp(t, regexp.MustCompile("^-----BEGIN PUBLIC KEY-----.*"), pub)
```

Comment:
I think an additional statement where you encode and then decode something should be added.
This is just to show the integrity of the keys, even though I believe the crypto package already should take care of that.

Reply:
more tests have been added